pyap_beauhurst/parser.py

```diff
@@ -81,14 +79,33 @@ def _parse_address(self, match: Union[re.Match, str]) -> Optional[address.Addres
                 cleaned_dict["match_start"] = match.start()
                 cleaned_dict["match_end"] = match.end()
                 # create object containing results
-                return address.Address(**cleaned_dict)
+                return address.Address(
```

Comment:
is this a pydantic model? Wouldn't it be easier to just do `address.Address.model_validate(cleaned_dict)`